packages/init/src/SliceMachineInitProcess.ts

```diff
@@ -198,106 +198,38 @@ export class SliceMachineInitProcess {
 		console.log(
 			`\n${chalk.bgGreen(` ${chalk.bold.white("Slice Machine")} `)} ${chalk.dim(
 				"→",
-			)} Initialization successful!`,
+			)} Initialization successful!
+			
+Continue with next steps in Slice Machine.
+`,
 		);
 
 		try {
-			const runSmCommand = this.context.projectInitialization?.patchedScript
-				? await getRunScriptCommand({
-						agent: this.context.packageManager || "npm",
-						script: "slicemachine",
-				  })
-				: await getExecuteCommand({
-						agent: this.context.packageManager || "npm",
-						script: "start-slicemachine",
-				  });
-
-			const runProjectCommand = await getRunScriptCommand({
-				agent: this.context.packageManager || "npm",
-				script: "dev",
-			});
-
-			const apiEndpoints = this.manager.getAPIEndpoints();
-			const wroomHost = new URL(apiEndpoints.PrismicWroom).host;
-
-			const dashboardURL = new URL(
-				`https://${this.context.repository.domain}.${wroomHost}`,
-			)
-				.toString()
-				.replace(/\/$/, "");
-			const apiURL = new URL(
-				"./api/v2",
-				`https://${this.context.repository.domain}.cdn.${wroomHost}`,
-			).toString();
-
-			// We prefer to manually allow console logs despite the app being a CLI to catch wild/unwanted console logs better
-			// eslint-disable-next-line no-console
-			console.log(`
-  YOUR REPOSITORY
-    Page Builder         ${chalk.cyan(dashboardURL)}
-    API                  ${chalk.cyan(apiURL)}
-
-  RESOURCES
-    Documentation        ${chalk.cyan(
-			this.context.framework.prismicDocumentation,
-		)}
-    Getting help         ${chalk.cyan("https://community.prismic.io")}
-
-  GETTING STARTED
-    Run Slice Machine    ${chalk.cyan(runSmCommand)}
-    Run your project     ${chalk.cyan(runProjectCommand)}
-	`);
-
-			if (this.options.starter && this.options.repository) {
-				const { openDashboard } = await prompt<boolean, "openDashboard">({
-					type: "confirm",
-					name: "openDashboard",
-					message: "Would you like to open your repository?",
-					initial: true,
-				});
+			if (this.options.startSlicemachine) {
```

Comment:
I think we added that to be consistent but I don't think anyone is using that, WDYT?
For me if it's used it could be something related to CI, and so no need to display the command, no?

Reply:
OK, fair enough yes! That's what I assumed.